lib/node_modules/@stdlib/assert/is-alphanumeric/test/test.js

```diff
@@ -0,0 +1,125 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2018 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isAlphaNumeric = require( './../lib' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof isAlphaNumeric, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function returns `true` if provided a string containing all numeric digits', function test( t ) {
```

Comment:
Let's remove this test.

Reply:
It was a carryover from copying and pasting. Resolved! The value for that test was merged into the larger "true" tests.